### PR TITLE
Fix another branch finding issue

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

@teizenman

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -208,24 +209,29 @@ def get_remote_file(
...
                     'Authorization': f"Bearer {githhub_config.Credentials.TOKEN}",
                     'Accept': f'application/vnd.github.VERSION.raw',
-                }
+                })  # Sometime we need headers
+                if not res.ok:  # sometime we need param token
+                    res = github_request(params={'token': githhub_config.Credentials.TOKEN})
+                res.raise_for_status()
             else:
-                print_color(
+                raise DemistoException(
```
Comment (roysagi) [inline demisto_sdk/commands/common/tools.py:226]: this will stop the command run - do we want that?

File: demisto_sdk/commands/common/constants.py [hunk 2]
```diff
@@ -817,7 +817,7 @@ class GithubContentConfig:
     def __init__(self, repo_name: Optional[str] = None):
         if not repo_name:
             try:
-                urls = GitUtil().repo.remote().urls
+                urls = list(GitUtil().repo.remote().urls)
```
Comment (roysagi) [inline demisto_sdk/commands/common/constants.py:820]: doesn't `.urls` return a list already?

File: demisto_sdk/commands/common/tools.py [hunk 3]
```diff
@@ -208,24 +209,29 @@ def get_remote_file(
     tag = tag.replace('origin/', '').replace('demisto/', '')
 
     github_path = urljoin(github_config.CONTENT_GITHUB_LINK, tag, full_file_path)
-
+    github_request = partial(requests.get, github_path, verify=False, timeout=10)
```
Comment (roysagi) [inline demisto_sdk/commands/common/tools.py:212]: don't like partial here. the way it was before was way clearer please revert this change.

File: demisto_sdk/commands/common/constants.py (hunk 2 above)
Comment (jochman) [reply demisto_sdk/commands/common/constants.py:820]: a generator. 
So when you're trying to iterate over it again it fails.

File: demisto_sdk/commands/common/tools.py (hunk 3 above)
Comment (jochman) [reply demisto_sdk/commands/common/tools.py:212]: I think it's much more clearer 😉

File: demisto_sdk/commands/common/tools.py (hunk 1 above)
Comment (jochman) [reply demisto_sdk/commands/common/tools.py]: It'll not fail as it's in the outer exception. But I've dealt with it.

**Review by roysagi — APPROVED** [review]
